components/DeckPreview.tsx

```diff
@@ -17,7 +17,7 @@ const deckIdStyle: CSS.Properties = {
 const DeckPreview = ({ deck, images }: { deck: IDeck, images: any }) => {
   return (
     <Link href={`/deck/${deck.id}`}>
-      <div className="flex flex-col w-96 justify-between p-4 bg-background rounded-md shadow-xl cursor-pointer">
+      <div className="flex flex-col w-96 justify-between p-4 bg-background rounded-md shadow-xl cursor-pointer ease-in-out duration-200 transform hover:rotate-2 hover:scale-95">
```

Comment:
I think the scale looks great, but maybe the rotate should be a bit more subtle? Happy to be overruled though!

Reply:
I also want to echo thank you for taking the time to make a contribution @anderskravis, and appreciate the thought here ❤️ !

Personally, I would also echo @0xOlias on not being a huge fan of the current rotation behavior.

To provide some helpful alternatives, I was taking a peek on codepen for some inspiration and found a few cool examples of card interactions (which will be very cool when we start polishing the game play through), however, I wasn't able to find a good design treatment for a deck.

- [Beautiful card horizontal flip](https://codepen.io/jasonchan/pen/vLeqyB)
- [Card rotation in 3d space](https://codepen.io/zwattic/pen/mKGRYm)

I don't have a strong stance against this change, but would like to see a few different treatments if that's possible.